source/common/upstream/upstream_impl.cc

```diff
@@ -1595,6 +1595,9 @@ bool BaseDynamicClusterImpl::updateDynamicHostList(
       current_priority_hosts.size());
   // Keep track of hosts we're adding (or replacing)
   absl::flat_hash_set<std::string> new_hosts_for_current_priority(new_hosts.size());
+  // Keep track of hosts for which locality is changed.
+  absl::flat_hash_set<std::string> existing_hosts_for_current_priority_locality_changed(
```

Comment:
maybe remove `existing` from this since we it's somewhat explicit if the locality changed? `hosts_with_updated_locality_for_current_priority` maybe?

Reply:
done